src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/LightBoxMercuryComponentObject.java

```diff
@@ -73,5 +94,172 @@ public void verifyImageWasChanged(String imageOnePath, String imageTwoPath) {
 	public void verifyLightboxClosed() {
 		Assertion.assertFalse(checkIfElementOnPage(currentImage));
 	}
+	
+	public String currentImageSrcPath () {
+		return currentImage.getAttribute("src");
+	}
+	
+	public void verifySwiping (PerformTouchAction touchAction, String direction, int attempts) {
+		String currentImageSrc = getCurrentImagePath();
+		String nextImageSrc;
+		boolean imageChanged = false;
+		int startX = 70;
+		int endX = 20;
+		int duration = 300;
+		int waitAfter = 5000;
+		if (direction == "right") {
+			int temp = startX;
+			startX = endX;
+			endX = temp;
+		} else {
+			direction = "left";
```

Comment:
Changed:
if (direction == "right") {...}
To:
if ("right".equals(direction)) {...}